Due legal issue the unRAR part of 7z had been removed (boo#1077978, boo#1090515)

---
 completions/7z |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: bash-completion-2.11/completions/7z
===================================================================
--- bash-completion-2.11.orig/completions/7z
+++ bash-completion-2.11/completions/7z
@@ -76,7 +76,7 @@ _7z()
             else
                 COMPREPLY=($(compgen -P${cur:0:2} -W '7z apm arj bzip2 cab
                     chm cpio cramfs deb dmg elf fat flv gzip hfs iso lzh lzma
-                    lzma86 macho mbr mslz mub nsis ntfs pe ppmd rar rpm
+                    lzma86 macho mbr mslz mub nsis ntfs pe ppmd rpm
                     squashfs swf swfc tar udf vhd wim xar xz z zip' \
                     -- "${cur:2}"))
             fi
@@ -110,7 +110,7 @@ _7z()
         #     (assumption: extensions are all lowercase)
         [[ $mode == w ]] &&
             _filedir '@(7z|bz2|swf|?(g)tar|?(t)[bglx]z|tb?(z)2|wim)' ||
-            _filedir '@(7z|arj|bz2|cab|chm|cpio|deb|dmg|flv|gem|img|iso|lz[ah]|lzma?(86)|msi|pmd|[rx]ar|rpm|sw[fm]|?(g)tar|taz|?(t)[bglx]z|tb?(z)2|vhd|wim|Z)'
+            _filedir '@(7z|arj|bz2|cab|chm|cpio|deb|dmg|flv|gem|img|iso|lz[ah]|lzma?(86)|msi|pmd|xar|rpm|sw[fm]|?(g)tar|taz|?(t)[bglx]z|tb?(z)2|vhd|wim|Z)'
     else
         if [[ ${words[1]} == d ]]; then
             local IFS=$'\n'
